pkg/virtctl/vmexport/vmexport_test.go

```diff
@@ -112,14 +115,16 @@ var _ = Describe("vmexport", func() {
 			_, err := virtClient.ExportV1beta1().VirtualMachineExports(metav1.NamespaceDefault).Create(context.Background(), vme, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
-			err = runCreateCmd(setFlag(vmexport.PVC_FLAG, pvcName))
+			err = runCreateCmd(
+				setFlag(vmexport.PVC_FLAG, pvcName),
+			)
 			Expect(err).To(MatchError(ContainSubstring("VirtualMachineExport 'default/test-vme' already exists")))
 		})
 
 		It("VirtualMachineExport doesn't exist when using 'download' without source type", func() {
 			err := runDownloadCmd(
 				setFlag(vmexport.VOLUME_FLAG, volumeName),
-				setFlag(vmexport.OUTPUT_FLAG, filepath.Join(GinkgoT().TempDir(), "output.img")),
+				setFlag(vmexport.OUTPUT_FLAG, outputPath),
```

Comment:
nit: I see that it was `output.img` instead of `disk.img`.

Reply:
Really shouldn't matter what the name is. I decided to stick with `disk.img` everywhere.